### DISCUSSION TITLE
Support for Pydantic v2

### DISCUSSION THREAD

**Discussion by farzbood** [collerek/ormar#1150] [Polls]
Hi everyone,
Since **FastAPI (0.100.0)** released with support for **Pydantic v2**, which brings big improvments in app's **performance**, how about a version bump in **ormar** to be compatible with **Pydantic v2** specifications?
This is to call for this upgrade/bump to support **FastAPI 0.100.0** and **Pydantic v2**.
Happy development, PEACE V ;)

**Comment by LasseRegin** [comment]
This is being discussed in https://github.com/collerek/ormar/issues/1115 FYI.

**Reply by farzbood** [reply]
Thanks for mentioning, good luck with upgrade.
